.github/workflows/test.yml

```diff
@@ -44,7 +44,8 @@ jobs:
 
     - name: Full integration test
       run: |
-        ./runkmd.sh ci/travis/${RELEASE} ci/travis/repository.yml --workspace /tmp/kmd-ws --cache /tmp/kmd-cache --prefix /tmp/pfx --release travis --locations-config $(realpath ci/travis/locations.yml) --renamer rename.ul --pip boot/bintools/pip
+        which pip
+        ./runkmd.sh ci/travis/${RELEASE} ci/travis/repository.yml --workspace /tmp/kmd-ws --cache /tmp/kmd-cache --prefix /tmp/pfx --release travis --locations-config $(realpath ci/travis/locations.yml) --renamer rename.ul --pip $(which pip)
```

Comment:
This is very incorrect! The `pip` here comes from the Python environment `kmd` runs in. Not the target Python environment. Remove `--pip $(which pip)` since it defaults to `pip` in the code, and will find the `boot/bintools/pip` instead as it should.

Reply:
I am still investigating, hold your horse. But default `pip` here failed, so there's something wrong with the environment.

Reply:
Ok. I'll hold on to my horse for the time being.